.travis.yml

```diff
@@ -13,6 +13,19 @@ node_js:
 - '6'
 - '7'
 
+cache:
+  directories:
+    - node_modules
+
+install:
+  - npm install
+
+script:
+  - npm run-script coverage
+
+notifications:
+  email: false
+
 matrix:
   include:
   - node_js: '0.10'
```

Comment:
Since this is a server-side thing I am not sure why you want all these other browser tests.

Check out https://github.com/alexa-js/alexa-app-server/pull/66 for a .travis.yml that might make more sense.